docs/build_node_setup.md

```diff
@@ -0,0 +1,62 @@
+---
+id: build_node_setup
+title: Set up a development node
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+This section runs you through the process of setting up a local HydraDX node for development. 
+
+:::note
+Are you looking to set up a node for validation purposes? Please move to our [validator setup guide](/node_setup).
+:::
+
+## 01 Install dependencies
+
+To prepare a local HydraDX node for development, your machine needs to cover all dependencies for running a Substrate node. You will need to install a Rust developer environment and make sure that it is configured properly for compiling Substrate runtime code to the WebAssembly (Wasm) target.
+
+You can install and configure all dependencies manually following the [Substrate guide](https://substrate.dev/docs/en/knowledgebase/getting-started), or you could let this script do all the work for you:
+
+```bash
+$ curl https://getsubstrate.io -sSf | bash -s -- --fast
+$ source ~/.cargo/env
+```
+
+## 02 Build
+
+Build the Wasm and native execution environments:
+
+```bash
+# Fetch source of the latest stable release
+$ git clone https://github.com/galacticcouncil/HydraDX-node -b stable
+
+# Build the binary
+$ cd HydraDX-node/
+$ cargo build --release
+```
+
+You should be able to find the build under `./target/release/hydra-dx`.
+
+## 03 Run
+
+Before running your build you can purge any existing development chains on your node:
```

Comment:
```suggestion
Before running your build you can purge any existing development chains on your machine:
```
I would say "on your machine" because chain data isn't stored inside the node directory.

Reply:
This only applies if you ran dev mode before and you are making some changes. If you're developing, you'll probably do this often.